packages/app/src/providers/mock/mock-file.ts

```diff
@@ -52,6 +54,26 @@ export function makeMockFile(): GroupWithChildren {
             }),
           ],
         }),
+        scalar('scalar_array', [1, 2], {
+          type: arrayType(intType(), [2]),
+          attributes: [
+            scalarAttr('attr', [1, 2], { type: arrayType(intType(), [2]) }),
+          ],
+        }),
+        scalar('scalar_vlen', [1, 2, 3], {
+          type: arrayType(intType()),
+          attributes: [
+            scalarAttr('attr', [1, 2, 3], { type: arrayType(intType()) }),
+          ],
+        }),
+        scalar('scalar_enum', 2, {
+          type: enumType(intType(8, true), { FOO: 0, BAR: 1, BAZ: 2 }),
+          attributes: [
+            scalarAttr('attr', 2, {
+              type: enumType(intType(8, true), { FOO: 0, BAR: 1, BAZ: 2 }),
+            }),
+          ],
+        }),
```

Comment:
Adding some mock scalar datasets to test `ArrayType` (with fixed or variable length), and `EnumType`.